.github/workflows/pull-request-relabelling.yaml

```diff
@@ -0,0 +1,23 @@
+name: Relabelling

```

Comment:
make it start with `r` to match rest of workflows in this project, same for line 7